packages/keychain/src/wallet/signer.ts

```diff
@@ -20,20 +21,26 @@ interface ContractCallOptions {
   functionArgs: ClarityValue[];
   version: TransactionVersion;
   nonce: number;
+  postConditions?: PostCondition[];
+  postConditionMode?: PostConditionMode;
 }
 
 interface ContractDeployOptions {
   contractName: string;
   codeBody: string;
   version: TransactionVersion;
   nonce: number;
+  postConditions?: PostCondition[];
+  postConditionMode?: PostConditionMode;
 }
 
 interface STXTransferOptions {
   recipient: string;
   amount: string;
   memo?: string;
   nonce: number;
+  postConditions?: PostCondition[];
+  postConditionMode?: PostConditionMode;
```

Comment:
Doesn't pc mode always have to be present, even when not including any conditions? 🤔 

Looks like in your uses it can be defined too, as you're setting the default.

Reply:
Yeah - this kinda follows the stacks-transactions API, which has a default mode. However, I am setting it to default "Allow", while stacks-transactions defaults to "Deny". I mainly kept that because I already had a default. In the spirit of #408 , I'll switch to not providing a default, which will then use the library default of Deny.

Reply:
Updated to remove any default setting 👍